### ISSUE TITLE
Datagrid: value with a tab character is trimmed.

### ISSUE THREAD

**Issue by ryanChrstn** [infor-design/enterprise#7849]
**Describe the bug**
A data with a tab character being passed to a datagrid, is trimmed when displayed.

**To Reproduce**

On a Datagrid component, create a dataset with a tab character on its value.
For example: edit the sample data from https://[latest-enterprise.demo.design.infor.com/api/datagrid-sample-data](https://latest-enterprise.demo.design.infor.com/api/datagrid-sample-data).

_From:_
{"id":"1","productId":**"T100"**,"sku":"#9000001-237","productName":"Compressor","activity":"Assemble Paint","quantity":1,"inStock":true,"price":210.99,"percent":0.1,"status":"OK","orderDate":"2018-08-07T06:00:00.000Z","action":"Action"}

_To:_
{"id":"1","productId":**"T100			"**,"sku":"#9000001-237","productName":"Compressor","activity":"Assemble Paint","quantity":1,"inStock":true,"price":210.99,"percent":0.1,"status":"OK","orderDate":"2018-08-07T06:00:00.000Z","action":"Action"}

After initializing the Datagrid, inspect the value of T100, it should have trailing tab characters.

Here's a sample of where I edited the data, to have tab characters on the first-four product id values.
https://jsfiddle.net/vhLto79x/

**Steps to reproduce the behavior:**
Prerequisite: Data passed on the Datagrid should have a tab character.
1. Initialize the Datagrid

**Expected behavior**
The value passed on the Datagrid should be matched on the displayed value, especially with values that have tab characters.

**Version**

- ids-enterprise: v4.82.0-dev

**Screenshots**
![image](https://github.com/infor-design/enterprise/assets/14357059/bb77b1ca-f966-4993-8544-a7a0bba73874)

**Platform**
- Infor Application/Team Name: Lawson
- OS Version: Windows 11
- Browser Name: Chrome
- Browser Version: Version 116.0.5845.97 (Official Build) (64-bit)

**Comment by tmcconechy** [comment]
I dont think we need to escape tab characters https://stackoverflow.com/questions/5105143/list-of-all-characters-that-should-be-escaped-before-put-in-to-regex

But im curious how this looks if we dont? Does it indent a bit?

TASK: Check escaping routine and remove tab key from logic

**Comment by tmcconechy** [comment]
I think there is another issue similar but i cant find it. For the fix add a setting on the column that lets you set `white-space`
